connectivity-exporter/Makefile

```
@@ -2,8 +2,49 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
+REGISTRY ?= ghcr.io
+IMAGE_NAME ?= gardener/connectivity-monitor
+IMAGE_TAG ?= test
+
 all: build
 
+.PHONY: docker/build
+docker/build:
+	docker build -t $(REGISTRY)/$(IMAGE_NAME):$(IMAGE_TAG) -f Dockerfile .
+
+.PHONY: docker/push
+docker/push:
+	docker push $(REGISTRY)/$(IMAGE_NAME):$(IMAGE_TAG)
+
+.PHONY: helm/generate
+helm/generate:
+	helm template \
+		connectivity-monitor ../charts/connectivity-monitor \
+		--output-dir bin/ \
+		--create-namespace \
+		--namespace connectivity-monitor \
+		--values ../charts/connectivity-monitor/values.yaml \
+		--set-string image.registry=$(REGISTRY) \
+		--set-string image.name=$(IMAGE_NAME) \
+		--set-string image.tag=$(IMAGE_TAG)
+
+.PHONY: helm/install
+helm/install:
+	helm install \
```

```suggestion
	helm upgrade --install \
```
With this you can run `make helm/install` to upgrade the version without having to `uninstall` first. The installation will also work the first time if the is no release yet.